appengine/pom.xml

```diff
@@ -35,6 +35,17 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
       </plugin>
+      <plugin>
+      	<groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+	      <configuration>
```

Comment:
Please fix the indentation here to 2 spaces.

Reply:
Thanks, got it!